### PR TITLE
TESB416-242 Infrastructure for Cinema Scraping

### PR DESCRIPTION
**Pull request by flitzpiepe96**
Implemented the infrastructure to easily add more cinema scrapers. This branch contains the bare infrastructure, for a working example (first scraper implemented) head over to TESB416-247

### CODE REVIEW

File: data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java [hunk 1]
```diff
@@ -103,12 +104,13 @@ private static void jettyEnableInherit(Server server) {
     @Bean
     public CommandLineRunner loadData(DataInitializer initializer) {
         return (args) -> {
-            initializer.initialize();
+            // initializer.initialize();
```
Comment (rgcjonas) [inline data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java]: We need to discuss a way to start the data server with fake data for local UI development where real data is not needed. This is closely related to the implementation of a persistent database.
If we decide to completely get rid of the fake data we should also remove the associated code and infrastructure.

File: data retrieval/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Fri Oct 07 22:06:00 CEST 2016
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
```
Comment (flitzpiepe96) [inline data retrieval/gradle/wrapper/gradle-wrapper.properties]: I had developed this project before the split was completed so it was copied over. I deleted the other unnecessary files, but this seems to have splipped through.

File: data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java [hunk 3]
```diff
@@ -103,12 +104,13 @@ private static void jettyEnableInherit(Server server) {
...
         };
     }
 
     public static void main(String[] args) {
-        // HACK: Passing java system properties is hard, environment variables are easy
+        // HACK: Passing java system properties is hard, environment variables
+        // are easy
```
Comment (flitzpiepe96) [inline data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java]: I dont know why... I though my formatting settings were correct. Let me check that

File: data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java (hunk 1 above)
Comment (flitzpiepe96) [inline data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java]: This was actually commited by accident. I turned this off for testing. I did not intend to turn off fake data just yet.

File: data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java [hunk 4]
```diff
@@ -66,7 +66,8 @@ public void run() {
         }
 
         @Override
-        public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler)
+                throws Exception {
```
Comment (rgcjonas) [inline data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java]: @flitzpiepe96  you forgot that one

**Pull request merged**